packages/editor/api-report.md

```diff
@@ -921,32 +948,35 @@ export class Editor extends EventEmitter<TLEventMap> {
     sendToBack(shapes: TLShape[]): this;
     // (undocumented)
     sendToBack(ids: TLShapeId[]): this;
-    setCamera(x: number, y: number, z?: number, { stopFollowing }?: TLViewportOptions): this;
+    setCamera(x: number, y: number, z?: number): this;
     // (undocumented)
-    setCroppingId(id: null | TLShapeId): this;
-    setCurrentPage(page: TLPage, opts?: TLViewportOptions): this;
+    setCroppingShapeId(id: null | TLShapeId): this;
+    setCurrentPage(page: TLPage): this;
     // (undocumented)
-    setCurrentPage(pageId: TLPageId, opts?: TLViewportOptions): this;
+    setCurrentPage(pageId: TLPageId): this;
     setCurrentTool(id: string, info?: {}): this;
     // (undocumented)
-    setEditingId(id: null | TLShapeId): this;
+    setEditingShapeId(id: null | TLShapeId): this;
     // (undocumented)
-    setErasingIds(ids: TLShapeId[]): this;
+    setErasingShapeIds(ids: TLShapeId[]): this;
     // (undocumented)
-    setFocusedGroupId(next: TLPageId | TLShapeId): this;
+    setFocusedGroupId(id: TLPageId | TLShapeId): this;
     // (undocumented)
-    setHintingIds(ids: TLShapeId[]): this;
+    setHintingShapeIds(ids: TLShapeId[]): this;
     // (undocumented)
-    setHoveredId(id: null | TLShapeId): this;
-    setOpacity(opacity: number, ephemeral?: boolean, squashing?: boolean): this;
+    setHoveredShapeId(id: null | TLShapeId): this;
+    setOpacity(opacity: number, opts?: {
```

Comment:
This implementation actually changed—we no longer set the opacity of children of groups, but groups themselves.